src/components/App.jsx

```diff
@@ -20,7 +20,7 @@ class App extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    if (this.props.user != nextProps.user) {
+    if (nextProps.user && nextProps.user != this.props.user) {
```

Comment:
Is the `!=` intentional, or is this supposed to be `!==` ?

Reply:
Oh, no it probably needs to be `!==`. Good catch.